### PR TITLE
Submission of Report for Topic 05

### PR DESCRIPTION
**Pull request by khsafkatamin**
I am Submitting my report of Traffic Light Detection from ACDC-RP 2023. 

- I have Added the report and results in reports section
- Added Task description of Topic 05 in the topics section 
- Added my contribution in the README section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ You can find all published research reports in the [`reports/`](https://github.c
 
 - [Domain Adaptation for Semantic Point Cloud Segmentation](https://github.com/ika-rwth-aachen/acdc-research-projects/tree/main/reports/02-Domain-Adaptation-for-Semantic-Point-Cloud-Segmentation/2022-09_marbuco/report.ipynb) by [@marbuco](https://github.com/marbuco/) (09/2022)
 - [Processing of Traffic Light Status Information in MPC-Planner](https://github.com/ika-rwth-aachen/acdc-research-projects/tree/main/reports/10-Processing-of-Traffic-Light-Status-Information-in-MPC-Planner/2023-09_MA_Leong_Hubbertz/report.ipynb) by [Lingyue MA](https://github.com/ihrer73), [Yi Hong Leong](https://github.com/malao98) and [Michael Hubbertz](https://github.com/EcxaByte) (09/2023)
+- [Traffic-Light-Detection](https://github.com/ika-rwth-aachen/acdc-research-projects/tree/main/reports/05-Traffic-Light-Detection/2023-09_Amin/report.ipynb) by [Kh Safkat Amin](https://github.com/khsafkatamin/) (09/2023)
```
Comment (lreiher) [inline README.md]:
```suggestion
- [Traffic Light Detection](https://github.com/ika-rwth-aachen/acdc-research-projects/tree/main/reports/05-Traffic-Light-Detection/2023-09_Amin/report.ipynb) by [Kh Safkat Amin](https://github.com/khsafkatamin/) (09/2023)
```

File: reports/05-Traffic-Light-Detection/2023-09_Amin/experiments/Ex-2/weights/best.pt
Comment (lreiher) [inline reports/05-Traffic-Light-Detection/2023-09_Amin/experiments/Ex-2/weights/best.pt:1]: The `.pt` files are pretty large. I believe that right now they are not yet stored in Git LFS. Can you please add them to Git LFS?

First you need to track `.pt` in Git LFS

```
git lfs track "*.pt"
```

However, I think that this doesn't move files to Git LFS that have already been committed. Would you mind checking yourself how to do that?

**Review by lreiher — CHANGES_REQUESTED** [review]
Thank you very much, I would be happy to merge this soon! Can you please try to move the .pt-files to Git LFS?

File: reports/05-Traffic-Light-Detection/2023-09_Amin/experiments/Ex-2/weights/best.pt
Comment (khsafkatamin) [reply reports/05-Traffic-Light-Detection/2023-09_Amin/experiments/Ex-2/weights/best.pt:1]: Hi, thanks for mentioning. I have added the .pt files with Git LFS and confirmed the tracked .pt file with `git lfs ls-files`. 

I have also made the suggested README changes.

**Pull request merged**